src/Components/Ignitor/src/Ignitor.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
This can likely be simplified in a lot of places using `$(DefaultNetCoreTargetFramework)` for instance. Just keeping it like this first to ensure validity.

Reply:
It should be fine. At some point this was hardcoded to 3.1 but we did not use the new property when we removed that contraint.

Reply:
Please do not leave hard-coded `net6.0` bits lying around. It just causes friction when we need to update the TFM -- and the next update will come in about a year.

Reply:
Introduced `$(TargetTFM)`. Is there any benefit in having `TargetTFM` **AND** `DefaultNetCoreTargetFramework` when they're both pointing to `net5.0`?

Reply:
Why did you add `$(TargetTFM)`❔ I don't see the point.

Reply:
It was the suggested var in https://github.com/dotnet/aspnetcore/issues/22737, but didn't see the point either as it duplicated `DefaultNetCoreTargetFramework`. Removing...